distribution/mlflow/overlays/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: mlflow
+resources:
+- ../../base
```

Comment:
What is the purpose of this overlay?

Reply:
Sorry, I hadn't pushed my last two commits. Should be gone now

Reply:
Now the `kustomization.yaml` in the mlflow repo seems to be missing completely.

Reply:
I removed it. The mlflow folder now just contains basic specs without any overlays so there is also no need to inject kustomize inbetween.